ipapython/ipautil.py

```diff
@@ -243,6 +243,21 @@ def set_ip_net(self, ifnet):
         self._net = ifnet
 
 
+class CheckedIPAddressLoopback(CheckedIPAddress):
+    """IPv4 or IPv6 address with additional constraints.
+
+    Reserved or link-local addresses are never accepted.
+    """
+    def __init__(self, addr, parse_netmask=True, allow_multicast=False):
+
+        super(CheckedIPAddressLoopback, self).__init__(
+                addr, parse_netmask=parse_netmask, allow_loopback=True,
+                allow_multicast=allow_multicast)
+
+        if self.is_loopback():
+            root_logger.warning("you're using a loopback ip")
```

Comment:
Actually, I want to see the IP Address in warning msg (there might be multiple addresses in IPv4 world)